cmd/ocm-support/delete/capabilities/account_capability/cmd.go

```diff
@@ -0,0 +1,59 @@
+package account
+
+import (
+	"fmt"
+
+	"github.com/openshift-online/ocm-cli/pkg/ocm"
+	"github.com/spf13/cobra"
+
+	"github.com/openshift-online/ocm-support-cli/pkg/account"
+	"github.com/openshift-online/ocm-support-cli/pkg/capability"
+)
+
+// CmdDeleteAccountCapability represents the delete account capability command
+var CmdDeleteAccountCapability = &cobra.Command{
+	Use:   "accountCapability [accountID] [capability]",
+	Short: "Removed a Capability to an Account",
+	Long:  "Removed a Capability to an Account",
```

Comment:
Maybe this is was a find/replace - I think we can re-write this. Please review the short/and long description of delete commands.